src/openrct2/Limits.h

```diff
@@ -0,0 +1,80 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2021 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+#include "rct2/Limits.h"
+
+namespace OpenRCT2::Limits
+{
+    constexpr const uint16_t MaxRidesInPark = 1000;
+    constexpr const uint16_t MaxStationsPerRide = 255;
+    // constexpr const uint16_t RideMeasurementMaxItems = 4800; // Is part of Measurement struct
+    using RCT12::Limits::CustomerHistorySize;
+    using RCT12::Limits::MaxGolfHoles;
+    using RCT12::Limits::MaxHelices;
+    using RCT12::Limits::MaxInversions;
```

Comment:
I'm not a fan doing that, that still ties our internal limits to the import of rct1/rct2, they should be separate constants.

Reply:
I want it to be clear if a constant has changed from RCT12 or not. If its changed then you would have constexpr const ... but otherwise its just a using.

Reply:
I know but that assumes we use the same internal limit which I want to be explicit about rather. The RCT12 stuff is primarily useful for the legacy importer/exporters. Changing that constant will will either break our structures or the importer so I don't know if its actually sensible.

Reply:
This look fine to me. It shows that these values are the same, and OpenRCT2::Limits is not used for importing SV6 or structure sizes afaik. I'd take this over magic numbers that are the same as RCT2's. What alternative do you propose?

Reply:
I would just declare it explicitly like 
```cpp
constexpr const uint16_t CustomerHistorySize = RCT12::Limits::CustomerHistorySize;
...
```
If we ever get to change numbers we just have to modify the assignment, either way I'm don't want to be in the way over this, can be changed if needed at any point.